projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -4703,6 +4703,187 @@ public string FatherIndustry
+        /// </example>
+        [Property("MotherHeight", Property.Types.Int32, "Mother Prenatal", "Mother's Height.", false, BFDR.IGURL.ObservationMotherHeight, true, 134)]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='8302-2')", "")]
+        public int? MotherHeight
+        {
+            get
+            {
+              var entry = Bundle.Entry.Where(e => e.Resource is Observation obs && CodeableConceptToDict(obs.Code)["code"] == "8302-2").FirstOrDefault();
+              if (entry != null)
+              {
+                  Observation observation = (Observation)entry.Resource;
+                  return (int?)(observation?.Value as Hl7.Fhir.Model.Quantity)?.Value;
+              }
+            return null;
+            }
+
+            set => SetHeight(value);
+        }
+
+        private string GetHeightEditFlagHelper()
+        {
+            Dictionary<string, string> editFlag = GetWeightEditFlag("8302-2");
+            if (editFlag.ContainsKey("code"))
+            {
+                string flagCode = editFlag["code"];
+                if (!String.IsNullOrWhiteSpace(flagCode))
+                {
+                    return flagCode;
+                }
+            }
+            return null;
+        }
+
+        private Dictionary<string, string> GetHeightEditFlag()
+        {
+            string code = "8302-2";
+            var entry = Bundle.Entry.Where(e => e.Resource is Observation obs && CodeableConceptToDict(obs.Code)["code"] == code).FirstOrDefault();
+            if (entry != null)
+            {
+                Observation observation = (Observation)entry.Resource;
+                Extension extension = observation?.Value?.Extension.FirstOrDefault(ext => ext.Url == VRExtensionURLs.BypassEditFlag);
+                if (extension != null && extension.Value != null && extension.Value.GetType() == typeof(CodeableConcept))
+                {
+                    return CodeableConceptToDict((CodeableConcept)extension.Value);
+                }
+            }
+            return EmptyCodeableDict();
+        }
+
+        private void SetHeightEditFlagHelper(string editFlag, string section)
+        {
+            // TODO add validation of editFlag and automate code system extraction once ValueSets.cs is available
+            if (String.IsNullOrEmpty(editFlag))
+            {
+                SetHeightEditFlag(EmptyCodeDict(), section);
+                return;
+            }
+            Dictionary<string, string> dictionary = new Dictionary<string, string>
+            {
+                { "code", editFlag },
+                { "system", "http://hl7.org/fhir/us/vr-common-library/CodeSystem/CodeSystem-vr-edit-flags" }
+            };
+            SetHeightEditFlag(dictionary, section);
+        }
+
+        private void SetHeightEditFlag(Dictionary<string, string> value, string section)
```

Comment:
method gets used in `MotherHeightEditFlagHelper` - please let me know if there is a cleaner way of doing this in C#

Reply:
Since there aren't multiple fields that are using these Get and Set helpers, I think a pattern more like PluralityEditFlag makes sense here vs a pattern like EducationLevel which was used for both Mother and Father fields. It should simplify the code so that the EditFlagHelper sets the Edit flag with the SetCodeValue() helper.